tests/test_utils.py

```diff
@@ -271,27 +270,31 @@ class PlainRelatedResourceTypeSerializer(serializers.Serializer):
     assert get_related_resource_type(field) == output
 
 
-def test_get_included_serializers():
-    class IncludedSerializersModel(DJAModel):
-        self = models.ForeignKey("self", on_delete=models.CASCADE)
-        target = models.ForeignKey(ManyToManyTarget, on_delete=models.CASCADE)
-        other_target = models.ForeignKey(ManyToManyTarget, on_delete=models.CASCADE)
+class IncludedSerializersModel(DJAModel):
```

Comment:
Why are you moving the serializer out of the function. Is the suggested change not working anymore with inline serializers?

Reply:
It works, but not self-referencing inline serializers.
The issue was that `self` refers to the inline serializer, which needs to be looked up by it's path in the module, and it can't be found since it's in the function scope...
So we do have a regression here:
- self-referencing inline serializers won't work.